torch/_inductor/fx_passes/joint_graph.py

```diff
@@ -432,12 +433,94 @@ def constant_fold_uniform_value(gm: torch.fx.GraphModule):
         remove_redundant_views(gm)
 
 
+def canonicalize_quant_mapping(gm: torch.fx.GraphModule):
```

Comment:
This feels weird

Reply:
I am going to update this - to reviewers - let's skip this part because going to revise it in base commit. I am more looking for feedback on the API to users (test files) and API to developers (after this has occurred)